SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -0,0 +1,370 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialWorkerConnection.h"
+
+#include "Interop/SpatialOutputDevice.h"
+
+#include "SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.h"
+
+#include "CoreMinimal.h"
+
+#define WORKERCONNECTION_TEST(TestName) \
+	GDK_TEST(Core, SpatialWorkerConnection, TestName)
+
+using namespace SpatialGDK;
+
+namespace
+{
+bool bClientConnectionProcessed = false;
+bool bServerConnectionProcessed = false;
+double MAX_WAIT_TIME = 10.0;
```

Comment:
```suggestion
double const MAX_WAIT_TIME = 10.0;
```
Nit, unsure if this is actually valuable

Reply:
Added const